lib/util.py

```diff
@@ -410,7 +399,7 @@ def format_satoshis_plain(x, decimal_point = 8):
     return "{:.8f}".format(Decimal(x) / scale_factor).rstrip('0').rstrip('.')
 
 
-def format_satoshis(x, num_zeros=0, decimal_point=8, precision=None, is_diff=False, whitespaces=False):
+def format_satoshis(x, num_zeros=1, decimal_point=8, precision=None, is_diff=False, whitespaces=False):
```

Comment:
I'm pretty sure modifying the defaults to this function will cause external plugins and even other parts of electron cash to have different/changed behavior.  What is the rationale for this?  this function is called EVERYWHERE (including external plugins).  I'd vote for leaving it at num_zeros=0 unless you can justify it.

Reply:
good catch. i opened ticket for this

https://github.com/Electron-Cash/Electron-Cash/issues/884

Reply:
OK, let's continue the conversation there.